libs/compatibility-spy/src/index.ts

```diff
@@ -0,0 +1,103 @@
+import { writeFileSync } from "fs";
+import { resolve } from "path";
+import { IPlatform } from "@winglang/sdk/lib/platform";
+import { Construct } from "constructs";
+import { App } from "@winglang/sdk/lib/core";
+
+const PARENT_PROPERTIES: Set<string> = new Set([
+  "node",
+  "onLiftMap",
+  ...Object.getOwnPropertyNames(Construct),
+  ...Object.getOwnPropertyNames(Construct.prototype),
+]);
+
+export class Platform implements IPlatform {
+  public readonly target = "compatibility-spy";
```

Comment:
Right now we dont have any compatibility checking in the Platform system but this platform would be agnostic so we can just use a "*" (since I plan to have that support in the future)
```suggestion
  public readonly target = "*";
```